plugins/mediaPlayer/mediaPlayer.ts

```diff
@@ -84,9 +84,7 @@ export const initMediaPlayer = (
       nextStartPosition = 0;
     }
     const token = await getAccessToken();
-    activeMedia.value = new MediaTrack(
-      createMedia(authorizedUrl(url, token)),
-    );
+    activeMedia.value = new MediaTrack(createMedia(authorizedUrl(url, token)));
```

Comment:
It's the same code. The syntax is slightly different, but I can't find any change

Reply:
are you looking at only the latest commit?

Reply:
oops